app/questionnaire/rules.py

```diff
@@ -281,3 +290,30 @@ def get_list_count(list_store, list_name):
 
 def is_goto_rule(rule):
     return any(key in rule.get('goto', {}) for key in ('when', 'block', 'group'))
+
+
+def get_default_answer(answer_id, answer_store, schema):
```

Comment:
From the name this method sounds like it should be on the questionnaire schema object and would always return the default answer. The schema logic below seems like it might be better placed there.